src/Raven.Client/Documents/Subscriptions/DocumentSubscriptions.cs

```diff
@@ -192,7 +193,7 @@ public string Create<T>(SubscriptionCreationOptions<T> options, string database
                         if (IncludesUtil.RequiresQuotes(include, out var escapedInclude))
                             criteria.Query += $"'{escapedInclude}'";
                         else
-                            criteria.Query += include;
+                            criteria.Query += QueryToken.IsKeyword(include) ? $"'{include}'" : include;
```

Comment:
What if the field contains `'` alredy?

Reply:
It is handled here: https://github.com/ravendb/ravendb/blob/v5.2/src/Raven.Client/Documents/Session/IncludesUtil.cs#L140